### ISSUE TITLE
Prettify variables

### ISSUE THREAD

**Issue by simPod** [prisma/graphql-playground#958]
It would be nice to have a Prettify button for variables. 

Sometimes when copying output from somewhere it pastes like one line with 5km length. 

Currently there's only prettify button that prettifies queries only.

**Comment by sergehuber** [comment]
Yes prettify for variables would be really great, because once you get used to it for queries you want it everywhere :)

**Comment by simPod** [comment]
I usually spend some time pressing enter after each key value pair when copy pasting json variable string from elsewhere. 🤔 

This is classic:
![image](https://user-images.githubusercontent.com/327717/53571253-b755c580-3b68-11e9-9de0-b17772aa2ded.png)

**Comment by sergehuber** [comment]
Yes exactly especially with the autocompletion that also doesn’t support prettifying 

Basically having the same features as in operation editing would be fantastic 


T +41 22 361 3424

9 route des Jeunes | 1227 Acacias | Switzerland
jahia.com
SKYPE | LINKEDIN | TWITTER | VCARD
  

> JOIN OUR COMMUNITY to evaluate, get trained and to discover why Jahia is a leading User Experience Platform (UXP) for Digital Transformation.
